### PR TITLE
Closes #2799: Add ReaderView L10N comments [ci skip]

### LINKED ISSUE #2799: ReaderView: L10N string review
There were some strings added for ReaderView for the appearance controls that should be reviewed and be done async to the integration work.

Strings: https://github.com/mozilla-mobile/android-components/blob/master/components/feature/readerview/src/main/res/values/strings.xml

This is the only screen that has strings for the android component, the rest would land in the Fenix repository:

Screenshot_2019-04-18 Reader View - Appearance

cc: @Pike & @Delphine

### PR DESCRIPTION
**Pull request by jonalmeida**
### Pull Request checklist

- [ ] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/android-components/blob/master/docs/changelog.md) or does not need one
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features

### CODE REVIEW

File: components/feature/readerview/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -6,18 +6,32 @@
...
+    <!-- Accessible description for the font option -->
     <string name="mozac_feature_readerview_sans_serif_font_desc">Sans Serif font</string>
-    <string name="mozac_feature_readerview_serif_desc">Serif</string>
+    <!-- Name for one of the font options available to use -->
+    <string name="mozac_feature_readerview_serif_font">Serif</string>
+    <!-- Accessible description for the font option -->
     <string name="mozac_feature_readerview_serif_font_desc">Serif font</string>
-    <string name="mozac_feature_readerview_negative_sign">-</string>
+
+    <string name="mozac_feature_readerview_negative_sign" translatable="false">-</string>
```
Comment (Delphine) [inline components/feature/readerview/src/main/res/values/strings.xml]: translatable="false": this won't work with our tooling. This should go in a separate file, such as non_localizable_strings.xml (which exists for other components).

File: components/feature/readerview/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -6,18 +6,32 @@
...
+    <!-- Accessible description for decreasing the font size -->
     <string name="mozac_feature_readerview_font_size_decrease_desc">Font size decrease</string>
-    <string name="mozac_feature_readerview_positive_sign">+</string>
+
+    <string name="mozac_feature_readerview_positive_sign" translatable="false">+</string>
```
Comment (Delphine) [inline components/feature/readerview/src/main/res/values/strings.xml]: Same comment as above

File: components/feature/readerview/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -6,18 +6,32 @@
...
+    <!-- Accessible description for increasing the font size -->
     <string name="mozac_feature_readerview_font_size_increase_desc">Font size increase</string>
+    <!-- Color option for the background -->
     <string name="mozac_feature_readerview_dark">Dark</string>
+    <!-- Accessible description for the for color option -->
```
Comment (Delphine) [inline components/feature/readerview/src/main/res/values/strings.xml]: Nit: "for the color option"

File: components/feature/readerview/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -6,18 +6,32 @@
...
     <string name="mozac_feature_readerview_dark_color_scheme_desc">Dark color scheme</string>
+    <!-- Color option for the background -->
     <string name="mozac_feature_readerview_sephia">Sepia</string>
+    <!-- Accessible description for the for color option -->
```
Comment (Delphine) [inline components/feature/readerview/src/main/res/values/strings.xml]: Same comment as above

**Comment by jonalmeida** [comment]
@Delphine I've addressed your comments. Could you take one last peek at it again? Thanks!

**Comment by Pike** [comment]
Do you want to add readerview to `l10n.toml` as part of this issue or should we file a follow-up?

**Comment by jonalmeida** [comment]
> Do you want to add readerview to `l10n.toml` as part of this issue or should we file a follow-up?

Done! :)

**Pull request merged**